lib/Newsletter/Renderer/EscapeHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace MailPoet\Newsletter\Renderer;
+
+class EscapeHelper {
+  static function escapeHtmlText($string) {
+    return htmlspecialchars((string)$string, ENT_NOQUOTES, 'UTF-8');
+  }
+
+  static function escapeHtmlAttr($string) {
+    return htmlspecialchars((string)$string, ENT_QUOTES, 'UTF-8', true);
```

Comment:
The `true` is default. Some methods here have it explicitly some implicitly. I think we can remove it. Also below.